.github/workflows/build-and-push.yaml

```diff
@@ -0,0 +1,32 @@
+name: 'Swagger CLI Build release'
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build-and-push-image:
+    name: 'Build catalog docker image and push it to dockerhub'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: 'checkout code'
+        uses: actions/checkout@v2
+
+      - name: 'build the image'
+        run: |
+          make GIT_REPOSITORY=${GITHUB_REPOSITORY} \
+          GIT_SHA=${GITHUB_SHA} \
+          GIT_SHORT_SHA=${GITHUB_SHA::7} \
```

Comment:
Why are you passing these in as variables, I thought that as you declared them in the Makefile they are set correctly?